### PR TITLE
Patch the missing Monitoring metrics

### PR DESCRIPTION
**Pull request by thxCode**
**Problem:**
- Cannot see the workload/container/pod metrics as `ProjectMetricGraph` has been deleted.
- Although delete monitoring action from schema resources, UI still can
seen `enableMonitoring` action link

**Solution:**
- Cannot delete `ProjectMetricGraph` directly.
- Remove actions addition from `Formatter`

**Issue:**
- https://github.com/rancher/rancher/issues/19516
- https://github.com/rancher/rancher/issues/19528

### CODE REVIEW

File: pkg/controllers/user/monitoring/projectHandler.go [hunk 1]
```diff
@@ -64,17 +64,23 @@ func (ph *projectHandler) doRevertSync(project *mgmtv3.Project, clusterName stri
...
+		projectName := fmt.Sprintf("%s:%s", project.Spec.ClusterName, project.Name)
+
 		for _, graph := range preDefinedProjectGraph {
-			err := ph.app.cattleProjectGraphClient.DeleteNamespaced(project.Name, graph.Name, &metav1.DeleteOptions{})
-			if err != nil && !apierrors.IsNotFound(err) {
-				return errors.Wrap(err, "failed to withdraw metric expression")
+			newObj := graph.DeepCopy()
+			newObj.Namespace = project.Name
+			newObj.Spec.ProjectName = projectName
+			if _, err := ph.app.cattleProjectGraphClient.Create(newObj); err != nil && !apierrors.IsAlreadyExists(err) {
```
Comment (orangedeng) [inline pkg/controllers/user/monitoring/projectHandler.go]: I don't want to do this logic in `doRevertSync` function. Please create a new function for it.

**Comment by orangedeng** [comment]
LGTM

**Comment by alena1108** [comment]
Can you check Drone failures @thxCode @orangedeng. Looks like the tests fail in the project creation area

File: pkg/controllers/user/monitoring/projectHandler.go [hunk 2]
```diff
@@ -168,6 +142,29 @@ func (ph *projectHandler) doSync(project *mgmtv3.Project, clusterName string) er
 	return nil
 }
 
+// doProjectGraphsSync creates the metric graphs per project for cluster monitoring
+func (ph *projectHandler) doProjectGraphsSync(project *mgmtv3.Project) error {
+	_, err := mgmtv3.ProjectConditionMetricExpressionDeployed.DoUntilTrue(project, func() (runtime.Object, error) {
```
Comment (orangedeng) [inline pkg/controllers/user/monitoring/projectHandler.go]: Please run this `doProjectGraphsSync` after the condition `NamespaceBackedResource` and `ProjectConditionInitialRolesPopulated` are set to true.

**Comment by orangedeng** [comment]
LGTM

**Pull request merged**